mantidproject/mantid
# ISIS Reflectometry: Highlight processed rows

A request by scientists was to highlight rows and groups that have been successfully processed to give users a better idea of how much data has been processed than the progress bar alone can do. This PR implements this in the interface

The goal of the original issue was to highlight the row/group currently being processed. This has now been changed to highlight all successfully processed rows/groups instead.

**To test:**

- Code review
- Ensure new unit tests pass and are sensible
- Manual testing can be done like this:
  - Login to the ICAT catalog
  - Open the ISIS Reflectometry interface change instrument to `POLREF`
  - Enter RB number `1520249` and transfer all runs
  - Select some rows/groups and process:
     - Selected rows should be highlighted sequentially before the containing group is
     - You may need to hover your mouse over the interface to see groups highlighting

Fixes #19930.

**Release Notes have been updated** 

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
